packages/react-components/src/components/utils/SendBoxUtils.ts

```diff
@@ -217,9 +217,10 @@ export const removeBrokenImageContent = (content: string): string => {
       img.removeAttribute('class');
       img.removeAttribute('src');
       img.removeAttribute('data-ui-id');
-      img.style.width = '';
-      img.style.height = '';
     }
+    // Clear maxWidth and maxHeight styles so that they can set in the style attribute
```

Comment:
It looks like we are updating `width` and `height` instead of maxWidth and maxHeight (as in comment).
Could you also add more details for `so that they can set in the style attribute`part?